neural_compressor/torch/quantization/config.py

```diff
@@ -76,6 +80,10 @@ def __init__(
         enable_mse_search: bool = False,
         group_dim: int = 1,
         return_int: bool = False,
+        double_quant_dtype: str = "fp32",
```

Comment:
Please register the supported options here: https://github.com/intel/neural-compressor/blob/9150181bb2ab71201fbdb052fbcaa2aba18a090a/neural_compressor/torch/quantization/config.py#L117-L125

Reply:
done